setup.py

```diff
@@ -19,7 +19,7 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    install_requires=['tensorflow==1.15',
+    install_requires=['torch',
```

Comment:
`click` should be added in the dependencies 

It is used by dnnlib: https://github.com/NVlabs/stylegan2-ada-pytorch/blob/main/legacy.py#L9